server_admin/topics/sessions/revocation.adoc

```diff
@@ -1,12 +1,23 @@
 
 [[_revocation-policy]]
 
-=== Revocation policies
+=== Revoking active sessions
 [role="_abstract"]
 
-If your system is compromised, you can revoke all active sessions and access tokens by clicking the `Sessions` screen *Revocation* tab.
+If your system is compromised, you can revoke all active sessions and access tokens.
+
+.Procedure
+. Click *Sessions* in the menu.
+ifeval::[{project_community}==true]
+. From the *Actions* list, select *Sign out all active sessions*.
+endif::[]
+ifeval::[{project_product}==true]
+. Click the *Revocation* tab.
+endif::[]
 
 .Revocation
-image:{project_images}/revocation.png[]
+image:{project_images}/revocation.png[Revocation]
 
-You can specify a time and date where sessions or tokens issued before that time and date are invalid using this console. Click *Set to now* to set the policy to the current time and date. Click *Push* to push this revocation policy to any registered OIDC client with the {project_name} OIDC client adapter.
+. Specify a time and date where sessions or tokens issued before that time and date are invalid using this console. 
```

Comment:
When you do this, do you see the screen shown.  It may not apply unless I actually have other active sessions, but for me it just ends my sessions.

Reply:
@jonkoops The PR looks good, but I am not seeing what you are seeing.  I just get logged out.  I assume it is because I have only one active session. Does that make sense?

Reply:
Yes, that is correct. Since the admin session is also terminated this is what will happen.